umh-core/pkg/config/env.go

```diff
@@ -105,6 +106,16 @@ func LoadConfigWithEnvOverrides(ctx context.Context, configManager *FileConfigMa
 				FSMInstanceConfig: FSMInstanceConfig{
 					DesiredFSMState: "active", // Default desired state for Redpanda
 				},
+				RedpandaServiceConfig: redpandaserviceconfig.RedpandaServiceConfig{
+					Topic: redpandaserviceconfig.TopicConfig{
+						DefaultTopicRetentionMs:    604800000,
```

Comment:
Better to add a comment that we are targeting 7 days. 60480000 looks like a magic number on the first look